src/types.ts

```diff
@@ -14,6 +14,11 @@ export interface ILightningWalletUser {
   // FIXME: Add a type for user here.
   user: unknown
   logger: Logger
+  config: Config
+}
+
+export type Config = {
```

Comment:
Can this type have a more descriptive name? `WalletConfig` perhaps?

Reply:
This is "the" config. Do we still need to make this more descriptive despite that fact?